Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/CrashesContentPage.xaml.cs

```diff
@@ -242,7 +242,9 @@ private void TrackException(Exception e)
             }
             Properties.Clear();
             RefreshPropCount();
-            Crashes.TrackError(e, properties);
+            // TODO: uncomment this when API will be added.
+            Crashes.TrackError(e, properties /*, App.GetErrorAttachments() */);
```

Comment:
`App.GetErrorAttachments()` returns `IEnumerable<ErrorAttachmentLog>`. It needs to be converted into an array as `App.GetErrorAttachments().ToArray()`.